labs/lab-2-javascript-game/style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  background-color: #000000;
+  padding: 50px;
+}
+
+h1 {
+  font-size: 25pt;
+  font-family: 'VT323', monospace;
```

Comment:
Cute font choice!

Coulda gone up in `body {` instead of in all these individual rules, tho.